### PR TITLE
Migrate FFmpeg build to it's own Dockerfile

### PR DESCRIPTION
**Pull request by lijo-jose**

### CODE REVIEW

**Comment by karthikmuralidharan** [comment]
@lijo-jose CI seems to have failed. Can you please check ?

File: ffmpeg-docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+FROM        base AS release
+MAINTAINER  Julien Rottenberg <julien@rottenberg.info>
+
+ENV         LD_LIBRARY_PATH=/usr/local/lib
+
+COPY --from=build /usr/local /usr/local/
+
+# Let's make sure the app built correctly
+# Convenient to verify on https://hub.docker.com/r/jrottenberg/ffmpeg/builds/ console output
```
Comment (karthikmuralidharan) [inline ffmpeg-docker/Dockerfile]: end of line not added.

File: ffmpeg-docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+        --disable-libxvid \
+        --enable-libx264 \
+        --disable-nonfree \
+        --disable-openssl \
+        --disable-libfdk_aac \
+        --disable-libkvazaar \
+        --disable-libaom --extra-libs=-lpthread \
+        --enable-postproc \
+        --enable-small \
+        --enable-version3 \
```
Comment (karthikmuralidharan) [inline ffmpeg-docker/Dockerfile]: @lijo-jose do we need to do this ?

File: ffmpeg-docker/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+        ./configure \
+        --disable-debug \
+        --disable-doc \
+        --disable-ffplay \
+        --disable-avresample \
+        --enable-shared \
+        --disable-libopencore-amrnb \
+        --disable-libopencore-amrwb \
+        --enable-gpl \
+        --enable-libxcb \
```
Comment (karthikmuralidharan) [inline ffmpeg-docker/Dockerfile]: Is this required ?

File: ffmpeg-docker/Dockerfile (hunk 1 above)
Comment (karthikmuralidharan) [inline ffmpeg-docker/Dockerfile]: @lijo-jose please take one more pass at reviewing this Dockerfile. Once you're confirmed, I can move this file to the `etherlabsio/ffmpeg` repo and have docker build a public image for this.

File: ffmpeg-docker/Dockerfile (hunk 2 above)
Comment (lijo-jose) [reply ffmpeg-docker/Dockerfile]: --enable-version3 is not needed. Will try without this.

**Comment by lijo-jose** [comment]
> @lijo-jose CI seems to have failed. Can you please check ?

we need to push the image from ffmpeg-docker/Dockerfile to dockerhub and if we use that for the ./Dockerfile, i think CI will work.

**Pull request merged**